### ISSUE TITLE
root user creation

### ISSUE THREAD

**Issue by seanieb** [makaimc/fsp-deployment-guide#10]
Most VPS hosting providers don't provide the ability to create the root password like the way you can in Linode. A default install of Ubuntu Server wont even let you SSH in as the root user (you need to set up openssh-server too). It might be worth simplifying this initial part of the guide, adding more generalizable steps, as it's not trivial to jump from Linode to AWS, a local VM, etc.

**Comment by jkokenge** [comment]
I think this is a good idea too.  AWS is ubiquitous these days and adapting what is already an absolutely great tutorial and resource to AWS would be a cool addition.  I'd be willing to help out with this however I can as well.

**Comment by mattmakai** [comment]
@seanieb @jkokenge I agree other providers do it differently. I deliberately made the decision to make this an opinionated guide to make it easier for people to choose the specific route of getting a Linode VM. AWS is a completely different beast with EC2 instances & non-persistent storage. it'd have to be an entirely new guide to make that approachable for beginners. Will look into other VPS providers but I haven't used any other ones extensively (well, since Slicehost shut down) so I wouldn't want to recommend them unless I had a good experience. Hearing good things about Digital Ocean so that'd be an option for me to try. Do you have other recommendations?

**Comment by dastagg** [comment]
I'm using Digital Ocean.
The only difference I see: After you create the droplet, you HAVE to connect to the server using 'ssh root@blahblah' and change the initial password that they send you in an email.

Then put that new password in fabfile.py and everything else is the same.
